### DISCUSSION TITLE
Add several servers with multiple instances each

### DISCUSSION THREAD

**Discussion by djkiuru** [trimble-oss/dba-dash#964] [Q&A]
If I have several clusters (9 clusters with 2 servers each). Each servers has several SQL instances, at least 3 or more. do I have to add the dbadash database to each instance and add each db manually through the config tool? 

Thanks!

**Comment by DavidWiseman** [comment]
DBA Dash monitors SQL Server at the instance level so you don't need a connection for every database.  If you have multiple named instances on the same cluster, each will need a connection in the config tool.  The same DBA Dash service can be used to monitor many instances.
Also, you only need a single DBA Dash repository database to store the collected data for your instances (unless you want to have multiple repositories).  This is set on the destination connection.  

Hope this helps,

David
